frontend/src/common/components/naviconsidebar/NavIconHomeButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, IconButton, Tooltip } from "@mui/material";
+import { Home } from "@mui/icons-material";
+import { useNavigate } from "react-router-dom";
+import * as routes from "../../../routes/constants";
+import { useTranslation } from "react-i18next";
+
+/**
+ * Displays the navigation icon for Home on the NavIconSideBar
+ */
+export const NavIconHomeButton = () => {
+    
+  const navigate = useNavigate()
+  const translationPrefix = 'navigation.button-bar'
+  const { t } = useTranslation()
+
+  return (
+    <div className="nav-icon-home-button">
+      <Box sx={{ border: '1px solid #003366', backgroundColor: '#003366', width: '45px' }}>
```

Comment:
It's better to place styles into a designated .scss file, so in this case it would be under "frontend/src/common/components/naviconsidebar/NavIconHomeButton.scss". I prefer this over inline styles.